modules/lightmapper_rd/lightmapper_rd.cpp

```diff
@@ -1501,8 +1629,29 @@ LightmapperRD::BakeError LightmapperRD::bake(BakeQuality p_quality, bool p_use_d
 		}
 
 		{
-			SWAP(light_accum_tex, light_accum_tex2);
-			BakeError error = _denoise(rd, compute_shader, compute_base_uniform_set, push_constant, light_accum_tex2, normal_tex, light_accum_tex, p_denoiser_strength, atlas_size, atlas_slices, p_bake_sh, p_step_function);
+			BakeError error;
+
+			int denoiser = GLOBAL_GET("rendering/lightmapping/denoising/denoiser");
+			if (denoiser == 1) {
+				// OIDN (external).
+				String oidn_path = EDITOR_GET("filesystem/tools/oidn/oidn_path");
+				Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+
+				if (da->dir_exists(oidn_path)) {
+					if (OS::get_singleton()->get_name() == "Windows") {
+						oidn_path = oidn_path.path_join("oidnDenoise.exe");
+					} else {
+						oidn_path = oidn_path.path_join("oidnDenoise");
+					}
+				}
+
+				ERR_FAIL_COND_V_MSG(oidn_path.is_empty() || !da->file_exists(oidn_path), BAKE_ERROR_LIGHTMAP_CANT_PRE_BAKE_MESHES, TTR("OIDN denoiser is selected in the project settings, but no or invalid OIDN executable path is configured in the editor settings."));
```

Comment:
It would be good to validate this before starting the bake. It's a bit user hostile that they can select OIDN, start baking, and at the end of the bake you just see this:

![image](https://github.com/godotengine/godot/assets/4701338/97a224b6-dc27-4039-8528-0764b83ab6d3)